docs/DOCUMENTACION.md

```diff
@@ -14,15 +14,29 @@ A través de la web de la Junta de Andalucía podremos obtener los datos relativ
+    - [M2](https://github.com/MarinoFajardo/ControlaTuGasto/milestone/5): Con este milestone externo se pretende crear el código necesario para la implementación de la lógica de negocio necesaria para calcular la ruta más corta entre un lugar de origen y un destino. Tras el cálculo de esta ruta, se mostrará el resultado por pantalla.<br/>
+    - [M3](https://github.com/MarinoFajardo/ControlaTuGasto/milestone/6): Con este milestone externo se pretende crear el código necesario para la implementación de la lógica de negocio necesaria para calcular la ruta más óptima entre un lugar de origen y un destino , siendo esta ruta más óptima la que guarde la mejor relación entre precio y duración. Tras el cálculo de esta ruta, se mostrará el resultado por pantalla.<br/>
 
-En el primer milestone(M0) se pretende realizar una representación del modelo del problema. En este caso se creará la implementación de una clase llamada tramo y otra clase ruta, siendo una ruta una colección de tramos entre un origen y un destino. Una vez obtenida la representación del modelo, en los milestones M1,M2,M3 se creará un grafo de tramos que conecten el lugar de origen con el destino y se aplicará la lógica de negocio mediante un algoritmo greedy para obtener las rutas descritas en las historias de usuario. En el milestone M1 se buscará la ruta más barata, en el milestone M2 se buscará la ruta más corta y en el milestone M3 la ruta más óptima. En cada uno de estos milestones se devolverá al usuario la ruta generada por pantalla.
\ No newline at end of file
+En el primer milestone(M0) se pretende realizar una representación del modelo del problema. En este caso se creará la implementación de una clase llamada tramo y otra clase ruta, siendo una ruta una colección de tramos entre un origen y un destino. Una vez obtenida la representación del modelo, en los milestones M1,M2,M3 se creará un grafo de tramos que conecten el lugar de origen con el destino y se aplicará la lógica de negocio mediante un algoritmo greedy para obtener las rutas descritas en las historias de usuario. En el milestone M1 se buscará la ruta más barata, en el milestone M2 se buscará la ruta más corta y en el milestone M3 la ruta más óptima. En cada uno de estos milestones se devolverá al usuario la ruta generada por pantalla.
+
+## Gestor de Dependencias
+A la hora de la elegir el gestor de dependencias para la aplicación es necesario establecer los criterios que que se van a tener en cuenta para esta eleción. En este caso se van a buscar gestores de dependencias que nos proporcionen una alta velocidad a la hora de las instalaciones y que no nos generen ficheros innecesarios en la aplicación. Teniendo estos criterios en cuenta, los gestores de dependencias que más se ajustan a lo anterior son los siguientes:
+- NPM: Gestor de dependencias por defecto de NodeJS, no nos genera deuda técnica debido a que ya viene instalado aunque ofrece un rendimiento muy bajo con respecto a pnpm y yarn.
```

Comment:
No has dicho nada de la deuda técnica en los criterios. Hablas de "generar ficheros innecesarios". Por favor, ajustaros a los criterios que ponéis (y viceversa) y ponedlos con precisión.